packages/core/rendering/core-rendering-server-internal/src/get_apm_config.ts

```diff
@@ -45,7 +45,8 @@ export const getApmConfig = (requestPath: string) => {
       ...config,
       pageLoadTraceId: traceId,
       pageLoadSampled: sampled,
-      pageLoadSpanId: backendTransaction.ensureParentId(),
```

Comment:
Q: should we keep this one and use the other ones only as fallbacks?

Reply:
`pageLoadSpanId` inverses the relationship - the RUM page-load transaction becomes the parent of the transaction on the backend. I still don't get what the use case is for this (or maybe I forgot), to me it makes more sense to have the server transaction to render the HTML be the parent of the page-load transaction. What are your thoughts here?

Reply:
TBH, I'm not familiar with why we did this and what `ensureParentId()` does. I'm mostly concerned about breaking the current behavior (I'm not sure if it's already broken 😅).

What you're saying makes a lot of sense to me. However, as I was trying to understand what this parameter does, I found [this piece of documentation](https://www.elastic.co/guide/en/apm/agent/rum-js/current/distributed-tracing-guide.html#dynamic-html-doc) recommending the use of `ensureParentId` as the `pageLoadSpanId` on dynamically generated HTML.

Reply:
Unless anyone is depending on the fact that the RUM transaction is the parent of the backend transaction (I honestly can't think of a use case for this), I think we can safely change this. However, I've reverted this change so we don't have to wait until https://github.com/elastic/apm-agent-rum-js/pull/1521 is merged.

Reply:
Both serve different purpose, For all APM agents (before Synthetics existed) RUM transaction is always considered the root. 
- So whenever the HTML is rendered from the Server, the Original Navigation timing span `Requesting and Receiving the document` will include the timing spent on the backend server which captures the time between `requestStart` and `responseEnd`. This is the part where the `pageLoadSpanId` helps where we can tie the frontend navigation with the backend HTML response generation. 
- Post Synthetics world, we need a new way to keep the RUM transaction as a child of the Synthetics one since browser is started from the synthetics side (Lots of details are in this issue - https://github.com/elastic/synthetics/issues/265). This PR https://github.com/elastic/apm-agent-rum-js/pull/1521 solves this linking by creating a new option to override the RUM parent trace. 

Hope this helps.